core/cloudflow-spark-testkit/src/main/scala/cloudflow/spark/testkit/SparkScalaTestSupport.scala

```diff
@@ -21,7 +21,12 @@ import org.scalatest._
 
 trait SparkScalaTestSupport extends WordSpec with MustMatchers with BeforeAndAfterAll {
 
-  val session: SparkSession = SparkSession.builder().master("local[2]").appName("test").getOrCreate()
+  val session: SparkSession = SparkSession
+    .builder()
+    .master("local[*]")
```

Comment:
Was it a cpu core issue?

Reply:
SO have been my guide: https://stackoverflow.com/a/52134851/7898052

Reply:
Cool.